docs/reference/release-notes/850.md

```diff
@@ -0,0 +1,188 @@
+
+### Simple browser notification
+
+Tasklist users can now receive a browser notification when new tasks are assigned to them. This requires Tasklist to run in the background.
+
+### HTML Viewer Component
+
+Developers can now add an [HTML viewer component](/components/modeler/forms/form-element-library/forms-element-library-html.md) to forms to render custom HTML with CSS. The HTML can be templated with process/form variables using expressions, which are sanitized to prevent XSS and clickjacking attacks. The CSS is scoped to the component, and does not impact the remaining form or embedding application.
+
+### Reports on Task Assignment
+
+The first iteration of this feature brings back existing features from Camunda 7 to Camunda 8. It is now possible to aggregate the duration of User Tasks in the states “Assigned” and “Unassigned” for an “User Task” / “Assignee”. The evaluation per “Assignee” can be switched off via a configuration. See the new options in the User Task Report.
+
+### Zeebe REST API infrastructure to build user task API
+
+The first step to offer an intuitive and consistent experience via a single, [unified Camunda 8 REST API](https://camunda.com/blog/2024/03/streamlining-camunda-apis-zeebe-rest-api/) is to provide Zeebe REST API. With this release, developers can use Zeebe REST API to manage [Zeebe User Tasks](/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md), enabling immediate task state changes. Zeebe REST API includes support for identity authentication and multi-tenancy, ensuring parity to Zeebe gRPC API.
+
+### Refactoring Suggestions
+
+Introducing AI-driven Refactoring Suggestions in our modeling canvas, designed to refine your BPMN models based on task descriptions. Enter a description, like 'Send a message', and immediately receive a practical suggestion, such as implementing it through a Slack connector. This feature aims to streamline your workflow, offering intuitive, actionable advice for transforming descriptions into efficient BPMN tasks.
+
+### Frontend Developer Documentation
+
+Developers now find [documentation](/apis-tools/frontend-development/01-task-applications/01-introduction-to-task-applications.md) about how to build custom frontends using Camunda's APIs and tools, including form-js.
+
+### KPI Report Wizard Framework
+
+The KPI Wizard is the new way to build a KPI Report quickly. The Wizard already contains predefined KPIs that can be selected. It guides the users through the needed customization and allows them to activate an alert that allows passive monitoring. The list of predefined KPIs will be extended with upcoming releases. Start setting up a first Process KPI now.
+
+### Use Secrets in Hybrid Mode
+
+You are now able to consume the secrets from the console in your own custom connector which is connected to our SAAS platform.
+
+### Multi-Region: Productise the 2 Regions, Active-Active, Self-Managed setup (Zeebe, Operate, Tasklist)
```

Comment:
I think this header is confusing me a bit. Maybe "Run Camunda in two regions synchronously with a multi-region setup"?

Reply:
I agree, although I think the Active-Active language is something we use in customer conversations. I'll make a note to follow up on this.

Reply:
Follow up here - https://camunda.slack.com/archives/C02AWA0RF8A/p1713198528757339